pom.xml

```diff
@@ -502,6 +502,9 @@
               <artifactId>${main.signature.artifact}</artifactId>
               <version>1.0</version>
             </signature>
+            <ignores>
+              <ignore>java.util.concurrent.ForkJoin*</ignore>
```

Comment:
yep that or there is an annotation I think we use elsewhere (in Platform.java). Might be better to use the annotation

Reply:
From gitter `@IgnoreJRERequirement`